ci-operator/config/openshift/ironic-image/openshift-ironic-image-release-4.2.yaml

```diff
@@ -0,0 +1,34 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base-os
+  to: ironic-image
+promotion:
```

Comment:
I think one of these needs `disable: true`.  I'd put it in the release-4.2 config based on another example.

Reply:
yep, added!

Reply:
correct, currently the master branch builds images promoted to `release-4.2`, so it should be disabled here